Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -326,7 +310,7 @@ void doPublishOdom() {
   double vth = (vr - vl)/(2 * TRACK_RADIUS);
   
   // Calculate the deltas since
-  double dt = current_time.toSec() - last_odom_publish_time.toSec();
+  double dt = current_time.toSec() - last_raw_odom_publish_time.toSec();
```

Comment:
Nevermind.
http://wiki.ros.org/CppStyleGuide#Static_class_variables

A better thing would be to not have globals, encapsulate things into classes, etc... but let's leave any such planning to the future, if we need to.
Don't do the static variable.